python/ray/dataframe/dataframe.py

```diff
@@ -2805,7 +2866,7 @@ def from_pandas(df, npartitions=None, chunksize=None, sort=True):
     """
 
     if npartitions is not None:
-        chunksize = int(len(df) / npartitions)
+        chunksize = int(len(df) / npartitions) + 1
```

Comment:
Slightly nicer to do this is `len(df) // npartitions + 1` I think, but not important